scss/_accordion.scss

```diff
@@ -85,6 +85,10 @@
   border-width: var(--#{$prefix}accordion-border-width) 0 0; // Boosted mod
 }
 
+.accordion-collapse {
+  padding: 5px 0;
```

Comment:
I think we should have `padding-top` and `padding-bottom` instead, if the user need some padding on x. Furthermore, I think that we should use a CSS variable. 
Have you tried to make the changes on `.accordion-body` (I forgot we had some already existing CSS variables)? 
Need design review?
Do we need padding on y and not only on top ?